client/src/modules/purchases/create/PurchaseOrderForm.js

```diff
@@ -91,26 +91,32 @@ function PurchaseOrderFormService(Inventory, AppCache, Store, Pool, PurchaseOrde
 
     // TODO(@jniles) - what do we do about user_id?  Should we
     // modify it?  Keep a table of modifications to purchase orders?
+    // NOTE(@jniles) - Vanga would like to see who modified the purchase orders.
```

Comment:
Is there an issue for this?

Reply:
No, I didn't have internet at the time of writing.  Now I do and here is the issue:  https://github.com/IMA-WorldHealth/bhima/issues/5721